.github/workflows/autotest.yml

```
@@ -62,7 +62,7 @@ jobs:
           architecture: ${{ matrix.architecture }}
       - name: install comtypes
         run: |
-          pip install --upgrade setuptools
+          pip install --upgrade setuptools build
```

With PEP517 build isolation and `setuptools` specified in your `[build-system].requires` pre-installing setuptools here should do nothing.
```suggestion
          pip install --upgrade build
```